rse_competencies_toolkit/settings/_production.py

```diff
@@ -17,3 +17,9 @@
 SECURE_HSTS_SECONDS = 15552000
 USE_X_FORWARDED_HOST = True
 EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+EMAIL_HOST = "smtp.sendgrid.net"
+EMAIL_PORT = 587
+EMAIL_USE_TLS = True
+EMAIL_HOST_USER = "apikey"
+EMAIL_HOST_PASSWORD = os.getenv("SENDGRID_API_KEY")
+DEFAULT_FROM_EMAIL = os.getenv("DEFAULT_FROM_EMAIL")
```

Comment:
The preference is that these are implemented in a separate deployment server-specific settings file that imports `_production.py` which is meant to be server-agnostic

Reply:
Thanks! I've just pushed some changes that should address this.